src/MainWindow.vala

```diff
@@ -54,7 +54,7 @@ public class MainWindow : Gtk.Dialog {
         Object (application: application,
                 icon_name: "com.github.rickybas.date-countdown",
                 resizable: false,
-                title: "Date Countdown");
+                title: (_("Date Countdown")));
```

Comment:
```suggestion
                title: _("Date Countdown"));
```

I guess there is an extra couple of brackets.

Reply:
Sure!